### ISSUE TITLE
Still crashing on Startup, I have all the required installations

### ISSUE THREAD

**Issue by CHAMPLOOx** [soulsmods/DSMapStudio#889]
I have all of the requirements and even optimized DSMapStudio.exe for NVIDIA and AMD and it still didn't work.

**Comment by Philiquaz** [comment]
I can't do much with this report. There's simply no information to work with, no crash log, nothing.
Your statement that you "optimized DSMapStudio.exe for NVIDIA and AMD" doesn't fill me with confidence that you're not making more problems.
Have you been able to run the low requirements exe?

**Comment by CHAMPLOOx** [comment]
> I can't do much with this report. There's simply no information to work with, no crash log, nothing. Your statement that you "optimized DSMapStudio.exe for NVIDIA and AMD" doesn't fill me with confidence that you're not making more problems. Have you been able to run the low requirements exe?

Yes, I have been able to use Low Requirement. How do I find the crashlogs?

**Comment by CHAMPLOOx** [comment]
> I can't do much with this report. There's simply no information to work with, no crash log, nothing. Your statement that you "optimized DSMapStudio.exe for NVIDIA and AMD" doesn't fill me with confidence that you're not making more problems. Have you been able to run the low requirements exe?
![image](https://github.com/soulsmods/DSMapStudio/assets/170052063/1f510469-7a3c-4d47-8156-6748ea1c855a)

**Comment by CHAMPLOOx** [comment]
> I can't do much with this report. There's simply no information to work with, no crash log, nothing. Your statement that you "optimized DSMapStudio.exe for NVIDIA and AMD" doesn't fill me with confidence that you're not making more problems. Have you been able to run the low requirements exe?

How deo I find the crash logs??

**Comment by CHAMPLOOx** [comment]
Thank you for your response.
